### PR TITLE
Fix issue #64

### PR DESCRIPTION
**Pull request by kakurasan**
This fixes issue #64.

unpatched:
unpatched
patched:
patched

### CODE REVIEW

File: bin/dnfdragora [hunk 1]
```diff
@@ -12,7 +12,6 @@
 
 import sys
 import gettext
-from gettext import gettext as _
```
Comment (Conan-Kudo) [inline bin/dnfdragora]: Don't we need this for the translated strings? We usually have our strings prefixed with `_`, like the following:

```python
parser.add_argument('--exit',        help=_('force dnfdaemon dbus services used by dnfdragora to exit'), action='store_true')
```

File: dnfdragora/ui.py [hunk 2]
```diff
@@ -29,7 +29,6 @@
 from dnfdragora import const
 
 import gettext
-from gettext import gettext as _, ngettext
```
Comment (Conan-Kudo) [inline dnfdragora/ui.py]: Not sure it's a good idea to get rid of this import.

File: dnfdragora/updater.py [hunk 3]
```diff
@@ -13,7 +13,6 @@
 
 from PIL import Image
 from dnfdragora import config, misc, ui
-from gettext import gettext as _
```
Comment (Conan-Kudo) [inline dnfdragora/updater.py]: Not sure it's a good idea to get rid of this import.

File: dnfdragora/progress_ui.py [hunk 4]
```diff
@@ -5,8 +5,6 @@
 
 import dnf.callback
 
-from gettext import gettext as _
```
Comment (Conan-Kudo) [inline dnfdragora/progress_ui.py]: Not sure it's a good idea to get rid of this import.

File: dnfdragora/groupicons.py [hunk 5]
```diff
@@ -1,5 +1,3 @@
-from gettext import gettext as _
```
Comment (Conan-Kudo) [inline dnfdragora/groupicons.py]: Not sure it's a good idea to get rid of this import.

File: dnfdragora/dnf_backend.py [hunk 6]
```diff
@@ -20,8 +20,6 @@
 import dnfdragora.const as const
 from dnfdragora.misc import ExceptionHandler, TimeFunction
 
-from gettext import gettext as _
```
Comment (Conan-Kudo) [inline dnfdragora/dnf_backend.py]: Not sure it's a good idea to get rid of this import.

File: dnfdragora/dialogs.py [hunk 7]
```diff
@@ -18,7 +18,6 @@
 from dnfdragora import const
 
 import gettext
-from gettext import gettext as _
```
Comment (Conan-Kudo) [inline dnfdragora/dialogs.py]: Not sure it's a good idea to get rid of this import.

File: dnfdragora/const.py [hunk 8]
```diff
@@ -17,8 +17,6 @@
 
 import hawkey
 
-from gettext import gettext as _
```
Comment (Conan-Kudo) [inline dnfdragora/const.py]: Not sure it's a good idea to get rid of this import.

File: dnfdragora/compsicons.py [hunk 9]
```diff
@@ -1,4 +1,3 @@
-from gettext import gettext as _
```
Comment (Conan-Kudo) [inline dnfdragora/compsicons.py]: Not sure it's a good idea to get rid of this import.

File: dnfdragora/basedragora.py [hunk 10]
```diff
@@ -23,7 +23,6 @@
 import dnfdragora.dialogs as dialogs
 import dnfdragora.dnf_backend
 import yui
-from gettext import gettext as _
```
Comment (Conan-Kudo) [inline dnfdragora/basedragora.py]: Not sure it's a good idea to get rid of this import.

**Comment by kakurasan** [comment]
The convenient function [gettext.install()](https://docs.python.org/3/library/gettext.html#gettext.install) *installs* the function `_()` (and optionally `ngettext()`) in Python's *built-in namespace*, so the `from gettext import gettext as _` codes *are not necessary anymore if `gettext.install()` is used*.

Example:
```python
#!/usr/bin/python3

import gettext

if __name__ == '__main__':
    gettext.install('dnfdragora', localedir='/usr/share/locale', names=('ngettext',))

    # gettext.gettext()
    translated_str = _('start using yui GTK+ plugin implementation')
    print('_("start using yui GTK+ plugin implementation") -> "{0}"'.format(translated_str))

    # gettext.ngettext()
    translated_str = ngettext('Downloading error', 'Downloading errors', 1)
    print('ngettext("Downloading error", "Downloading errors", 1) -> "{0}"'.format(translated_str))
    translated_str = ngettext('Downloading error', 'Downloading errors', 2)
    print('ngettext("Downloading error", "Downloading errors", 2) -> "{0}"'.format(translated_str))
```

**Comment by Conan-Kudo** [comment]
@kakurasan Neat! 👍

**Comment by Conan-Kudo** [comment]
@kakurasan Thanks for the changes, I've merged them! 👍

**Pull request merged**